From 9984dc0728a084cc1516d89bc711ea6fe9552d68 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Thu, 23 Feb 2023 11:41:49 +0100
Subject: [PATCH] skip running bootcmd mfg

when booting with uuu, because we want to run balenaOS

Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 common/autoboot.c | 16 +---------------
 1 file changed, 1 insertion(+), 15 deletions(-)

diff --git a/common/autoboot.c b/common/autoboot.c
index 2802e381ad..ebc7f7369b 100644
--- a/common/autoboot.c
+++ b/common/autoboot.c
@@ -349,18 +349,7 @@ const char *bootdelay_process(void)
 						  bootdelay);
 
 #if defined(is_boot_from_usb)
-	if (is_boot_from_usb() && env_get("bootcmd_mfg")) {
-		disconnect_from_pc();
-		printf("Boot from USB for mfgtools\n");
-		bootdelay = 0;
-		env_set_default("Use default environment for \
-				 mfgtools\n", 0);
-	} else if (is_boot_from_usb()) {
-		printf("Boot from USB for uuu\n");
-		env_set("bootcmd", "fastboot 0");
-	} else {
 		printf("Normal Boot\n");
-	}
 #endif
 
 	debug("### main_loop entered: bootdelay=%d\n\n", bootdelay);
@@ -380,10 +369,7 @@ const char *bootdelay_process(void)
 		s = env_get("bootcmd");
 
 #if defined(is_boot_from_usb)
-	if (is_boot_from_usb() && env_get("bootcmd_mfg")) {
-		s = env_get("bootcmd_mfg");
-		printf("Run bootcmd_mfg: %s\n", s);
-	}
+		 printf("Skip running bootcmd_mfg: %s\n", s);
 #endif
 
 	if (IS_ENABLED(CONFIG_OF_CONTROL))
-- 
2.37.2

